python-backend/graph_utils.py

```diff
@@ -20,177 +22,97 @@ class Point2D(TypedDict):
     y: str
 
 
-def error_coordinates(values: dict[int, mpmath.mpf], limit: mpmath.mpf, iterations: int) -> list[Point2D]:
+async def chart_coordinates(values: dict[int, mpmath.mpf],
```

Comment:
This file used to comprise several routines which compute a series of values. Consolidated all of the routines into one so we could return a block of results across all three charts simultaneously to the UI via socket.